### ISSUE TITLE
Add :focus style for search bar

### ISSUE THREAD

**Issue by rhymes** [thepracticaldev/dev.to#4663]
**Is your feature request related to a problem? Please describe.**

In a11y terms, [focusable elements should have focus styling](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard?utm_source=devtools&utm_medium=a11y-panel-checks-keyboard#Focusable_element_must_have_focus_styling), meaning that there should be a visual feedback of when the search bar is focused.

This also helps in knowing if what you're typing is in the box, or finding inline in the page through browser's features or anything else.

**Describe the solution you'd like**

Don't know if this is the correct one, but I tried something like this:

![search bar](https://user-images.githubusercontent.com/146201/67878651-f135fc00-fb3b-11e9-8b1a-ca536358229b.gif)

**Comment by emimuresan** [comment]
Hi @rhymes , i can take a look at this and try to open a PR soon.

**Comment by maestromac** [comment]
Resolved with https://github.com/thepracticaldev/dev.to/pull/4668
